content/docs/using-pulumi/crossguard/snyk-policy.md

```diff
@@ -0,0 +1,256 @@
+---
+title_tag: "Snyk Container Scanning | CrossGuard"
+meta_desc: Snyk container scanning helps you ensure that your Docker images are secure.
+title: Synk Container Scanning
+h1: Snyk Container Scanning
+weight: 4
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+  usingpulumi:
+    parent: crossguard
+aliases:
+- /docs/guides/crossguard/snyk-container-scanning/
+---
+<!-- markdownlint-disable ul code -->
+
+## Overview
+
+[Snyk container scanning](https://github.com/pulumi/templates-policy/tree/master/snyk-typescript) is a Pulumi policy as code template that allows you to uses the [Snyk CLI](https://docs.snyk.io/scan-using-snyk/snyk-container/scan-container-images) to scan [`docker.Image`](/registry/packages/docker/api-docs/image/) resources for vulnerabilities.
```

Comment:
The links here seem like they may be doing to the wrong place. I'd probably expect the first link to point to general information about what Snyk container scanning is and the CLI link to point me to the CLI (although we seem to link to that in the prereqs, so maybe no need to link to anything here).

Reply:
You're right. I don't love the code as the first link, but the problem is that the policy pack itself is called "Snyk Container Scanning", and I feel like I have to link on the first mention. Suggestions welcome.